src/SonarScanner.Shim/SonarScanner.Shim.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net46</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
```

Comment:
Same question as above: this isn't an entry point so why do we need to target multiple frameworks?

Also, only one version of this assembly is included in the zip.